src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -838,7 +838,9 @@ internal static object Create(IHostEnvironment env, ModelLoadContext ctx, object
     internal static class CalibratorUtils
     {
         // maximum number of rows passed to the calibrator.
-        private const int _maxCalibrationExamples = 1000000;
+        // if 0, we'll actually look through the whole dataset to
+        // when training the calibrator
```

Comment:
Earlier, you had explained to me that if this value is zero, we would look through the whole dataset, but still only use a million rows (randomly selected) for calibration.  Is that correct?

Can you please clarify the exact behavior in comments?

Reply:
It is correct for PlattCalibrator, but depending of the calibrator the behavior is different. If this is 0, the only thing that _does_ happen for all the calibrators is that we'll look through all the dataset. I explained this further on the other comment I left, so I don't think it's necessary to clarify it more in here.